test/TestProjects/MgmtMockAndSample/specification/mockSwagger/examples/createVault.json

```diff
@@ -1,7 +1,6 @@
 {
   "parameters": {
     "resourceGroupName": "sample-resource-group",
-    "vaultName": "sample-vault",
```

Comment:
How about we duplicate this example, and remove this value in the new example, so that we could leave some comment on the name of the example to avoid future confusions

Reply:
updated